src/styles/index.scss

```
@@ -1,3 +1,119 @@
+$white: #fff;
+$lightBlue: #00acdc;
+$gray: #616070;
+$lightGray: #f3f3f3;
+$darkBlue: #060b35;
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $lightGray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $white;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin: 0 0 4px;
```

reset default margin for `h2`, `p` etc tags to avoid reseting this value each time
```suggestion
    margin-bottom: 4px;
```